core/src/main/java/org/mule/runtime/core/el/mvel/MVELExpressionLanguage.java

```diff
@@ -373,6 +373,19 @@ public String parse(String expression, final MuleEvent event, FlowConstruct flow
     }, expression);
   }
 
+  @Override
+  public String parse(String expression, final MuleEvent event, MuleEvent.Builder eventBuilder, FlowConstruct flowConstruct)
+      throws ExpressionRuntimeException {
```

Comment:
existing method should delegate to this one rather than copy/paste implemntation.